src/index.ts

```diff
@@ -870,6 +918,27 @@ class Datastore extends DatastoreRequest {
   Transaction = Transaction;
 }
 
+/*! Developer Documentation
+ *
+ * All async methods (except for streams) will return a Promise in the event
+ * that a callback is omitted.
+ */
+promisifyAll(Datastore, {
```

Comment:
could we use `util.promisify`?

Reply:
> could we use `util.promisify`?

@bcoe  for consistency `promisifyAll` is used as currently in all the library we are using the same.

Reply:
:+1: works for me, might be worth doing some work on the `util` module to make sure `promisfyAll` takes advantage of `util.promisify`.